QuestionAnswering/QA_SEQ2SEQ/data_generation/locate_then_ask/ontokin/rxn/ask.py

```diff
@@ -1,59 +1,51 @@
-import copy
 import random
-from locate_then_ask.data_model import AskDatum
-from locate_then_ask.ontokin.graph2sparql import OKGraph2Sparql
+from constants.functions import AggOp
+
+from locate_then_ask.ask import ask_name
+from locate_then_ask.graph2sparql import Graph2Sparql
 from locate_then_ask.query_graph import QueryGraph
 
 
 class OKReactionAsker:
-    def __init__(self):
-        self.graph2sparql = OKGraph2Sparql()
-
     def ask_name(self, query_graph: QueryGraph, verbalization: str):
-        query_graph = copy.deepcopy(query_graph)
-        query_graph.nodes["Reaction"]["question_node"] = True
-
-        verbalization = "What is " + verbalization
-        query_sparql = self.graph2sparql.convert(query_graph)
-
-        return AskDatum(
-            query_graph=query_graph,
-            query_sparql=query_sparql,
-            verbalization=verbalization,
-        )
+        return ask_name(query_graph, verbalization, "Reaction")
 
     def ask_count(self, query_graph: QueryGraph, verbalization: str):
         assert "Mechanism" in query_graph.nodes()
```

Comment:
See above comment.

Reply:
I've replaced the `assert` statement with `raise ValueError(...)`.